docs/content/tutorials/tutorial-compaction.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: doc_page
+---
+
+# Tutorial: Compacting segments
+
+This tutorial demonstrates how to compact existing segments into fewer but larger segments.
```

Comment:
Suggest adding a short blurb about why you would want to do this and how it can improve performance / memory usage

Reply:
Added a blurb about performance